tutorials/getting_started/so_you_built_a_model.md

```diff
@@ -0,0 +1,210 @@
+# So You Built a Model, Now What?
+
+After you've built and trained a model, it's likely that you want to use it
+for something. In this tutorial we'll cover both
+
+* How to make predictions using your model, and
+* How to run a web demonstration of your model
+
+Here we'll be working the S2 classification model
```

Comment:
I would call it the "paper classification model" rather than using an internal project code name.